defs/gmake.mk

```diff
@@ -357,6 +357,53 @@ ifneq ($(REVISION_IN_HEADER),$(REVISION_LATEST))
 $(srcdir)/revision.h: $(REVISION_H)
 endif
 
+# Show Cargo progress when doing `make V=1`
+CARGO_VERBOSE_0 = -q
+CARGO_VERBOSE_1 =
+CARGO_VERBOSE = $(CARGO_VERBOSE_$(V))
+
+.PHONY: yjit-static-lib
+yjit-static-lib:
```

Comment:
You can split `if` and `else` targets below by

```
yjit-static-lib: yjit-static-lib-$(YJIT_SUPPORT)
yjit-static-lib-release:
	echo 'building Rust YJIT (release mode)'
	$(RUSTC)...
yjit-static-lib-dev: 
	echo 'building Rust YJIT (dev mode)'
	$(RUSTC)...
```

Reply:
Thanks for this nice technique; it makes `make V=1` nicer! I added f04da3cf8798e25885a2383737b1e40a6d0cbeb3